src/test/java/com/shapesecurity/shift/es2017/parser/expressions/literals/LiteralRegExpExpressionTest.java

```diff
@@ -1,18 +1,239 @@
+        "/(?=t|v|X|.|$||)*/",
+        "/(?=t|v|X|.|$||)/",
+        "/(?=t|v|X|.|$||)/u",
+        "/./",
+        "/.|./",
+        "/.||./",
+        "/X$/",
+        "/X*/",
+        "/X*?/",
+        "/X+/",
+        "/X+?/",
+        "/X?/",
+        "/X??/",
+        "/X{5,10}/",
+        "/X{5,10}?/",
+        "/X{5,}/",
+        "/X{5,}?/",
+        "/X{5}/",
+        "/X{5}?/",
+        "/[-X]/",
+        "/[-X]/u",
+        "/[0-9-a-]/",
+        "/[0-9-a-]/u",
+        "/[0-9-a-z-]/",
+        "/[0-9-a-z-]/u",
+        "/[0-9-a-z]/",
+        "/[0-9-a-z]/u",
+        "/[X-]/",
+        "/[X-]/u",
+        "/[X]/",
+        "/[X]/u",
+        "/[\\-]/",
+        "/[\\-]/u",
+        "/[\\1-\\2]/",
+        "/[\\123]/",
+        "/[\\126-\\127]/",
+        "/[\\128-9]/",
+        "/[\\153]/",
+        "/[\\15]/",
+        "/[\\1]/",
+        "/[\\5]/",
+        "/[\\72]/",
+        "/[\\7]/",
+        "/[\\9]/",
+        "/[\\D]/",
+        "/[\\S]/",
+        "/[\\W]/",
+        "/[\\\\5]/",
+        "/[\\_]/",
+        "/[\\b]/",
+        "/[\\b]/u",
+        "/[\\c0-\\c9]/",
+        "/[\\c10]/",
+        "/[\\c5]/",
+        "/[\\c]/",
+        "/[\\ca]/",
+        "/[\\ca]/u",
+        "/[\\s-X]/",
+        "/[\\s]/",
+        "/[\\u{1F4A9}-\\u{1F4AB}]/u",
+        "/[\\w]/",
+        "/[]/",
+        "/[]/u",
+        "/[^-X]/",
+        "/[^-X]/u",
+        "/[^0-9-a-]/",
+        "/[^0-9-a-]/u",
+        "/[^0-9-a-z-]/",
+        "/[^0-9-a-z-]/u",
+        "/[^0-9-a-z]/",
+        "/[^0-9-a-z]/u",
+        "/[^X-]/",
+        "/[^X-]/u",
+        "/[^X]/",
+        "/[^X]/u",
+        "/[^]/",
+        "/[^]/u",
+        "/[{}[||)(()\\]?+*.$^]/",
+        "/[{}[||)(()\\]?+*.$^]/u",
+        "/[💩-💫]/u",
+        "/\\$/",
+        "/\\$/u",
+        "/\\0/",
+        "/\\0/u",
+        "/\\1()/",
+        "/\\1()/u",
+        "/\\123/",
+        "/\\2()()/u",
+        "/\\2()/",
+        "/\\2/",
+        "/\\BX/",
+        "/\\D/",
+        "/\\D/u",
+        "/\\L/",
+        "/\\S/",
+        "/\\S/u",
+        "/\\W/",
+        "/\\W/u",
+        "/\\bX/",
+        "/\\c/",
+        "/\\cZ/",
+        "/\\ca/",
+        "/\\d/",
+        "/\\d/u",
+        "/\\d]/",
+        "/\\f/",
+        "/\\k/",
+        "/\\n/",
+        "/\\r/",
+        "/\\s/",
+        "/\\s/u",
+        "/\\t/",
+        "/\\u10AB/",
+        "/\\u10AB/u",
+        "/\\uD800/u",
+        "/\\uD800\\uDF00/u",
+        "/\\uDF00/u",
+        "/\\ud800\\u1000/u",
+        "/\\u{001AD}/u",
+        "/\\u{0}/u",
+        "/\\u{10FFFF}/u",
+        "/\\u{10}/u",
+        "/\\v/",
+        "/\\w/",
+        "/\\w/u",
+        "/\\x0F/",
+        "/\\xAA/",
+        "/\\xZZ/",
+        "/]*/",
+        "/^$\\b\\B/",
+        "/^X/",
+        "/t{5/",
+        "/{5,10X}/",
+        "/{5,X}/",
+        "/{5.}/",
+        "/{dfwfdf}/",
+        "/|.||.|/",
+        "/|/",
+        "/}*/",
+        "/[\\99-\\98]/",
+        "/[\\99-\\100]/",
```

Comment:
Let's get the tests in [this commit](https://github.com/shapesecurity/shift-regexp-acceptor-js/commit/5f254de0eef1178de9f023b9ef9b10c530c1a7d8#diff-c1d9835a59f549d9ee3db0bfb144e125R187) over here as well.